### DISCUSSION TITLE
Will uBo also stop working at _some_ point in non-official Chrome builds, such as Thorium?

### DISCUSSION THREAD

**Discussion by convexshiba** [uBlockOrigin/uBlock-issues#3334] [Q&A]
title says it all. My guess is yes but not an expert. Thanks.

**Comment by uBlock-user** [comment]
Ask thorium dev, uBO cannot do anything about it.

**Comment by garry-ut99** [comment]
It's unclear what do you exactly mean by "at _some_ point", noone in the world knows in the advance whether any browser will support MV2 extensions forever.

Also it's browsers, not extensions, who decide whether to support MV2 extensions or not, MV2 extensions will work as long a browser supports them, hence a proper places to ask such questions are browser's issue tracker/forum, not extensions's.

Anyway since you already asked, there is Supremium (a Chromium-based browser fork) which is going to support MV2 even in 2025: 
- https://github.com/uBlockOrigin/uBlock-issues/discussions/3315

**Reply by convexshiba** [reply]
Thanks for the explanation. Yeah I was thinking where would be the right venue to ask this question and figured I'd just start here.

I've since asked in Thorium support and got an answer that they have no plan to deprecate MV2 even if/when official Chrome does in the future. 

https://github.com/Alex313031/thorium/discussions/767

**Reply by garry-ut99** [reply]
uBO too has no plans to deprecate [uBO](https://github.com/gorhill/uBlock) (MV2), it's still going to be maintentained, despite existence of [uBOL](https://github.com/uBlockOrigin/uBOL-home) (MV3), if this clears your doubts. Also good to hear that Thorium is going to support MV2. 

By the way you spammed their Discussions, as you should have been ask the question in already existing thread: 
- https://github.com/Alex313031/thorium/discussions/12

you also spammed uBO Discussions, as you should have been ask the question in one of already existing threads, for example:
- https://github.com/uBlockOrigin/uBlock-issues/discussions/2977
- https://github.com/uBlockOrigin/uBlock-issues/discussions/2987

You are not in your own house, avoid spamming trackers with duplicate threads in the future, as there is too much spam already, the more duplicated threads, the bigger the clutter.
